styleguide/derek/pattern-components/service-level-teaser/service-level-teaser.ejs

```diff
@@ -0,0 +1,102 @@
+<div class=" half-padding container">
+  <h3>Service Level Teasers (default)</h3>
+</div>
+
+<!-- Service Level Teaser -->
+<div class="half-padding">
+  <div class="container">
+    <div class="row">
+      <div class="col-sm-4">
+        <%- partial('_partials/teaser.ejs', {
+          title: "Human Experts",
+          icon: "idio-winner",
+          price: "$500/m"
+        }) %>
+      </div>
+      <div class="col-sm-4">
+        <%- partial('_partials/teaser.ejs', {
+          title: "Navigator",
+          icon: "srv-navigator",
+          price: "$600/m"
+        }) %>
+      </div>
+      <div class="col-sm-4">
+        <%- partial('_partials/teaser.ejs', {
+          title: "Aviator",
+          icon: "srv-aviator",
+          price: "$700/m"
+        }) %>
+      </div>
+    </div>
+    <div class="serviceLevelTeaser-bottom">
+      <a href="#" class="serviceLevelTeaser-bottomLink">Compare All</a>
+    </div>
+  </div>
+</div>
+
+<div class="half-padding container">
+  <h3>Service Level Teasers 2 only (default)</h3>
+</div>
+
+<!-- Teaser with only 2 option -->
+<div class="half-padding">
+  <div class="container">
```

Comment:
It's worth noting that either this PR, or #444 will need to be adjusted (depending on merge order), to allow for usage of the new `.derek-paddedContainer` class. It doesn't require immediate action, but we'll need to make sure we remember that come merge time.